frappe/website/website_generator.py

```diff
@@ -9,11 +9,10 @@
 
 
 class WebsiteGenerator(Document):
-	website = frappe._dict()
-
 	def __init__(self, *args, **kwargs):
 		self.route = None
 		super().__init__(*args, **kwargs)
+		self.website = frappe._dict()
```

Comment:
This is most likely a breaking change. All website generators who were specifying different ones will have to do it AFTER init. Either by modifying or overriding.